resources/logging/charts/fluentbit/values.yaml

```diff
@@ -315,3 +315,13 @@ tolerations:
 
 # Pod affinity constraint
 affinity: {}
+
+# external service entry configuration, this config add an external service to the service mesh
```

Comment:
that is a very istio specific configuration.. I thought we maybe can formulate it a bit more use case specific. So I want to enable a 3party integration. For that I might need to configure the Protocol settings explicit. Why the relevant key in the values.yaml is ServiceEntry? If I have no istio background I will not get this..